bundle/manifests/prometheus-sandboxed-containers-rules_monitoring.coreos.com_v1_prometheusrule.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: prometheus-sandboxed-containers-rules
+spec:
+  groups:
+  - name: kata_monitor_rules
+    rules:
+    - expr: sum(kata_monitor_running_shim_count)
+      record: cluster:kata_monitor_running_shim_count:sum
```

Comment:
Hi @littlejawa, can you please confirm if this yaml was generated by running `make bundle` after adding the `config/kata-monitor/kata-monitor-prometheus-rules.yaml` ?

Reply:
I'm not sure which command generated it, but I found it in my "bundle" folder after building / testing the operator.

Is it wrong?

Reply:
No it's not wrong, but I expected the bundle manifest and the config manifest to be exactly the same. So I was curious :-)

Reply:
Maybe what it is trying to tell me is that I should fix the indentation in the config manifest :-)
Also: it removes the namespace... As it's installed by the operator, maybe I don't need to specify it in the config manifest either? 

Note that I did the same (have the namespace specified) in the ServiceMonitor config manifest, and it was removed from the bundle too.